default.nix

```diff
@@ -0,0 +1,18 @@
+{ pkgs ? import <nixpkgs> { } }:
+let
+  mkdocs_ = with pkgs.python3Packages;
+    toPythonApplication (pkgs.mkdocs.passthru.overridePythonAttrs (prev: {
+      # propagatedBuildInputs = [ mkdocs-material ] ++ prev.propagatedBuildInputs;
+      propagatedBuildInputs = prev.propagatedBuildInputs;
+      # pythonPath = [ mkdocs-material ];
+      doCheck = true;
+    }));
```

Comment:
I'm struggling with this wrapper. If I add mkdocs-material, then the mkdocs binary starts failing like this:
```
./result/bin/mkdocs: line 7: /nix/store/rmp0ydgcrccnkzn4m5ggwivy1xw3igls-python3.10-mkdocs-1.4.2/bin/mkdocs: No such file or directory
```

Reply:
```

:b python3.pkgs.toPythonApplication (python3.pkgs.mkdocs.overridePythonAttrs (old: { propagatedBuildInputs = old.propagatedBuildInputs; }))
This derivation produced the following outputs:
  dist -> /nix/store/p1hzma8sam1r1nqkl7b98nss42n4chf6-python3.10-mkdocs-1.4.2-dist
  out -> /nix/store/s7mqhzir568n3cmi4wqjpj0nnbv9ib17-python3.10-mkdocs-1.4.2
╭─[~/git/nixpkgs]─[harmonia-switch]
╰─ % /nix/store/s7mqhzir568n3cmi4wqjpj0nnbv9ib17-python3.10-mkdocs-1.4.2/bin/mkdocs
Usage: mkdocs [OPTIONS] COMMAND [ARGS]...

  MkDocs - Project documentation with Markdown.

Options:
  -V, --version  Show the version and exit.
  -q, --quiet    Silence warnings
  -v, --verbose  Enable verbose output
  -h, --help     Show this message and exit.

Commands:
  build      Build the MkDocs documentation
  gh-deploy  Deploy your documentation to GitHub Pages
  new        Create a new MkDocs project
  serve      Run the builtin development server


```